packages/keystone/file/ConvertFileToTable.js

```diff
@@ -1,7 +1,7 @@
 const jschardet = require('jschardet')
 
+const { clearString, encode, detectEncoding, convertEncoding, ROWS_COUNT_LIMIT_EXCEEDED } = require('./constants')
```

Comment:
But functions still should be imported from utils. Isn't it?

Reply:
oops :)
sure